packages/element-lib/package.json

```diff
@@ -67,10 +71,13 @@
     "pouchdb-adapter-idb": "^7.1.1",
     "pouchdb-adapter-leveldb": "^7.1.1",
     "pouchdb-adapter-memory": "^7.1.1",
+    "prettier": "^2.0.5",
     "rxdb": "^8.5.0",
     "rxjs": "^6.5.2",
     "secp256k1": "^3.6.2",
-    "web3": "^1.2.2",
+    "truffle-assertions": "^0.9.2",
+    "typescript": "^3.9.7",
```

Comment:
this should be a dev dependency

Reply:
use `npm i <package> --save-dev` to automatically add it to devDependencies when installing. you can copy and paste it to correct the issue as is.

Reply:
done with new commit @gjgd